tests/helpers/compute_helpers.py

```diff
@@ -5,7 +5,7 @@
 from ocean_provider.constants import BaseURLs
 from ocean_provider.utils.accounts import sign_message
 from ocean_provider.utils.datatoken import get_dt_contract
-from tests.helpers.service_definitions import (
+from tests.helpers.ddo_dict_builders import (
```

Comment:
I noticed that `tests.helpers.service_definitions` doesn't exist, and the imported functions are in `tests.helpers.ddo_dict_builders` module, so I updated it. 
Do you have this problem in  `tests.helpers.ddo_dict_builders` @DMats ?